definitions/windows.yaml

```diff
@@ -135,6 +135,19 @@ sources:
 supported_os: [Windows]
 urls: ['https://msdn.microsoft.com/en-us/library/windows/desktop/ms678415(v=vs.85).aspx']
 ---
+name: InternetExplorerBrowserHelperObjects
+doc: Loaded on Internet Explorer startup
+sources:
+- type: REGISTRY_KEY
+  attributes:
+    keys:
+    - 'HKEY_LOCAL_MACHINE\Software\Microsoft\Windows\CurrentVersion\Explorer\Browser Helper Objects\*'
```

Comment:
Put this on a single line. Also it would be good if you could name your PR after the artifact you're adding so the commit history is more meaningful.